test/L2/paused/L2GovernorPaused.t.sol

```diff
@@ -0,0 +1,155 @@
+import { TimelockController } from "@openzeppelin/contracts/governance/TimelockController.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import { TimelockControllerUpgradeable } from
+    "@openzeppelin-upgradeable/contracts/governance/extensions/GovernorTimelockControlUpgradeable.sol";
+import { Test, console } from "forge-std/Test.sol";
+import { L2Governor } from "src/L2/L2Governor.sol";
+import { L2GovernorPaused } from "src/L2/paused/L2GovernorPaused.sol";
+import { Utils } from "script/contracts/Utils.sol";
+
+contract MockL2GovernorV2 is L2Governor {
+    function version() public pure virtual override returns (string memory) {
+        return "2.0.0";
+    }
+}
+
+contract L2GovernorPausedTest is Test {
+    Utils public utils;
+    L2Governor public l2GovernorImplementation;
+    L2Governor public l2Governor;
+
+    IVotes votingPower;
+    address[] executors;
+    TimelockController timelock;
+    address initialOwner;
+
+    function setUp() public {
+        utils = new Utils();
+
+        // set initial values
+        votingPower = IVotes(address(0x1));
+        executors.push(address(0)); // executor array contains address(0) such that anyone can execute proposals
+        timelock = new TimelockController(0, new address[](0), executors, address(this));
+        initialOwner = address(this);
+
+        console.log("L2GovernorTest address is: %s", address(this));
+
+        // deploy L2Governor Implementation contract
+        l2GovernorImplementation = new L2Governor();
+
+        // deploy L2Governor contract via proxy and initialize it at the same time
+        l2Governor = L2Governor(
+            payable(
+                address(
+                    new ERC1967Proxy(
+                        address(l2GovernorImplementation),
+                        abi.encodeWithSelector(l2Governor.initialize.selector, votingPower, timelock, initialOwner)
+                    )
+                )
+            )
+        );
+
+        assertEq(l2Governor.name(), "Lisk Governor");
+        assertEq(l2Governor.version(), "1.0.0");
+        assertEq(l2Governor.votingDelay(), 0);
+        assertEq(l2Governor.votingPeriod(), 604800);
+        assertEq(l2Governor.proposalThreshold(), 300_000 * 10 ** 18);
+        assertEq(l2Governor.timelock(), address(timelock));
+        assertEq(l2Governor.quorum(0), 24_000_000 * 10 ** 18);
+        assertEq(address(l2Governor.token()), address(votingPower));
+        assertEq(l2Governor.owner(), initialOwner);
+
+        // assure that address(0) is in executors role
+        assertEq(timelock.hasRole(timelock.EXECUTOR_ROLE(), address(0)), true);
+
+        // Upgrade from L2Governor to L2GovernorPaused
+        L2GovernorPaused l2GovernorPausedImplementation = new L2GovernorPaused();
+        l2Governor.upgradeToAndCall(address(l2GovernorPausedImplementation), "");
+        assertEq(l2Governor.version(), "1.0.0-paused");
```

Comment:
Wouldn't it be good to assert that everything else did not change? I mean:
```
        assertEq(l2Governor.name(), "Lisk Governor");
        assertEq(l2Governor.votingDelay(), 0);
        assertEq(l2Governor.votingPeriod(), 604800);
        assertEq(l2Governor.proposalThreshold(), 300_000 * 10 ** 18);
        assertEq(l2Governor.timelock(), address(timelock));
        assertEq(l2Governor.quorum(0), 24_000_000 * 10 ** 18);
        assertEq(address(l2Governor.token()), address(votingPower));
        assertEq(l2Governor.owner(), initialOwner);
```

Reply:
Assertion are added